SpatialGDK/Source/SpatialGDKFunctionalTests/Private/TestMaps/GeneratedTestMap.cpp

```diff
@@ -101,10 +104,12 @@ void UGeneratedTestMap::GenerateBaseMap()
 
 	// Default player start location is chosen so that players spawn on server 1 by default.
 	// Individual test maps can change this if necessary.
-	APlayerStart* PlayerStart = AddActorToLevel<APlayerStart>(CurrentLevel, FTransform(FVector(-100, -100, 100)));
-
-	// TODO: Maybe figure out how to set the default viewpoint when opening the map, so we don't start in the ground (maybe together with
-	// other viewing position issues), ticket: UNR-4975.
+	FTransform PlayerTransform;
+	FVector PlayerTranslation = FVector(-500, -500, 200);
+ 	PlayerTransform.SetTranslation(PlayerTranslation);
+	FRotator RotatorToOrigin = UKismetMathLibrary::MakeRotFromX(-PlayerTranslation);
```

Comment:
Seems a bit excessive to use Kismet here (I just pinched the implementation) and then the include can be removed 

```suggestion
	FRotator RotatorToOrigin = FRotationMatrix::MakeFromX(-PlayerTranslation).Rotator();
```

Reply:
Also maybe a small comment here, saying we want the player start to look at the origin (which is what this does).
If you want to be super-precise, you can also include a comment about the inability to take rotation around the Y axis (or whichever axis it was) into account.